libraries/botbuilder-azure/src/cosmosDbPartitionedStorage.ts

```diff
@@ -106,6 +127,22 @@ export class CosmosDbPartitionedStorage implements Storage {
         if (!cosmosDbStorageOptions.authKey) { throw new ReferenceError('authKey for CosmosDB is required.'); }
         if (!cosmosDbStorageOptions.databaseId) { throw new ReferenceError('databaseId is for CosmosDB required.'); }
         if (!cosmosDbStorageOptions.containerId) { throw new ReferenceError('containerId for CosmosDB is required.'); }
+        // In order to support collections previously restricted to max key length of 255, we default
+        // compatabilityMode to 'true'.  No compatibilityMode is opt-in only.
+        if (typeof cosmosDbStorageOptions.compatibilityMode === "undefined") {
+            cosmosDbStorageOptions.compatibilityMode = true;
+        }
+        if (cosmosDbStorageOptions.keySuffix) {
+            if (cosmosDbStorageOptions.compatibilityMode){
+                throw new ReferenceError('compatibilityMode cannot be true while using a keySuffix.');
+            }
+            // In order to reduce key complexity, we do not allow invalid characters in a KeySuffix
+            // If the keySuffix has invalid characters, the escaped key will not match
+            var suffixEscaped = CosmosDbKeyEscape.escapeKey(cosmosDbStorageOptions.keySuffix);
```

Comment:
```suggestion
            const suffixEscaped = CosmosDbKeyEscape.escapeKey(cosmosDbStorageOptions.keySuffix);
```

nitpicky, but might prevent somebody doing something weird with it in the future

Reply:
committed your suggestion ... thanks!